django_app/redbox_app/templates/homepage.html

```diff
@@ -20,9 +20,12 @@ <h1 class="govuk-heading-xl">Ask any question of documents in your Redbox</h1>
 <div class="govuk-width-container">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <p class="govuk-heading-m govuk-!-margin-top-5 govuk-!-padding-top-5">Control your box of documents</p>
-            <p class="govuk-heading-m govuk-!-margin-top-5 govuk-!-padding-top-5">Ask any question and AI will generate a response</p>
-            <p class="govuk-heading-m govuk-!-margin-top-5 govuk-!-padding-top-5">Use up to, and including, Official Sensitive documents</p>
+            <p class="govuk-heading-m govuk-!-margin-top-5 govuk-!-padding-top-5">Upload</p>
```

Comment:
Please can these headings be `<h2>`s. The reason I hadn't made them headings originally was because they didn't have content underneath them, so weren't technically headings at the time. But sorry for the false steer!